src/riscv-integration.adoc

```diff
@@ -506,25 +506,34 @@ interrupts cause the <<pcc>> to be set to the capability with
 its address incremented by four times the interrupt cause number.
 
 Capabilities written to <<mtvecc>> also include writing the MODE field in
-**mtvecc.address[1:0]**. As a result, a representability and sealing check is
-performed on the capability with the legalized (WARL) MODE field included in
-the address. The tag of the capability written to <<mtvecc>> is cleared if
-either check fails.
+**mtvecc.address[1:0]**, which is a WARL field, meaning that the capability address can
+be legalized. If the MODE field is non-zero after any legalization,
+then the address used for the exception vector (which has **mtvecc.address[1:0]=2'b00**)
+will not match the value read by software from the CSR.
+
+In this situation, the same capability can effectively have more than one address.
```

Comment:
```suggestion
```
I think this sentence is confusing and is best removed. We're basically just using this capability and with the address updated.